### DISCUSSION TITLE
Proxy apps that require client certs

### DISCUSSION THREAD

**Discussion by mukerjee** [gravitational/teleport#12201] [Q&A] [answered]
Hello, is the any way to proxy an internal app when the internal app requires a client cert?

Optimally one should be able to set a client cert + key and a CA cert for each application backend. From what I see in the docs the only options the app proxy has for TLS-based backends is `insecure_skip_verify`.

**Comment by webvictim** [comment, marked as answer]
This is not currently supported.

**Reply by mukerjee** [reply]
Thanks! does this happen to be on the roadmap?

**Reply by webvictim** [reply]
I don't believe so; please feel free to [raise a feature request](https://github.com/gravitational/teleport/issues/new?assignees=&labels=feature-request&template=feature_request.md) if it's something you'd like to see!
